ui/src/app/shared/services/view-preferences-service.ts

```diff
@@ -71,7 +71,7 @@ const DEFAULT_PREFERENCES: ViewPreferences = {
     version: 1,
     appDetails: {
         view: 'tree',
-        resourceFilter: ['kind:Deployment', 'kind:Service', 'kind:Pod', 'kind:StatefulSet', 'kind:Ingress', 'kind:ConfigMap', 'kind:Job', 'kind:DaemonSet', 'kind:Workflow'],
```

Comment:
this is a regression, the logic around defaults was complex, I suspect buggy, not useful, so I just removed it